app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/BusinessIdentificationController.scala

```diff
@@ -192,7 +208,8 @@ class BusinessIdentificationController @Inject()(
                        .completePartialSubscription(knownFacts.utr, knownFacts.postcode)
                        .map { arn =>
                          mark("Count-Subscription-PartialSubscriptionCompleted")
-                         Redirect(routes.SubscriptionController.showSubscriptionComplete()).flashing("arn" -> arn.value)
+                         Redirect(routes.SubscriptionController.showSubscriptionComplete())
+                           .flashing("arn" -> arn.value)
```

Comment:
Just noticed this arn in the flash scope. Shouldn't it be in the (encrypted) session instead? I think the subscription complete page is expecting it from there.